app/src/models/HighImpactActionRanking.ts

```diff
@@ -60,7 +60,7 @@ export class HighImpactActionRanking
         },
         jobId: {
           type: DataTypes.TEXT,
-          allowNull: true,
```

Comment:
We might need to create a migration to add this non-null constraint in the DB as well? Or is this already the case for the `job_id` column?